web/src/utils/getNavigationDeepLinks.ts

```diff
@@ -0,0 +1,21 @@
+import { LocationModel } from 'api-client'
+
+export const getNavigationDeepLinks = (location: LocationModel, fallback: string): string => {
```

Comment:
Could we somehow pass `isIos` and `isAndroid` as params and move the functionality to the api client? We implement the same on native, don't we?

Reply:
No react native has some interna function like Plattform.select

Reply:
you don't have to use it though, there is also `Platform.OS`: https://reactnative.dev/docs/platform#os
so you could just do the following:
``` ts
getNavigatonDeepLinks(location, fallback, Platform.OS === 'android', Platform.OS === 'ios')
```

Reply:
but we are on web here?

Reply:
Created https://issues.tuerantuer.org/browse/IGAPP-986